python/ray/_raylet.pyx

```diff
@@ -231,10 +303,139 @@ cdef class RayletClient:
     def disconnect(self):
         check_status(self.client.get().Disconnect())
 
-    def submit_task(self, Task task_spec):
-        check_status(self.client.get().SubmitTask(
-            task_spec.execution_dependencies.get()[0],
-            task_spec.task_spec.get()[0]))
+    def submit_task(self,
+                    worker,
+                    function_descriptor_list,
+                    function_signature,
+                    args,
+                    kwargs,
+                    TaskID current_task_id,
+                    put_function,
```

Comment:
Do we need to pass in `put_function`? Since we already pass in `worker`, can we just do `worker.put`?